content/en/data_jobs/airflow.md

```diff
@@ -241,7 +241,41 @@ Check that the OpenLineage environment variables are correctly set on the Astron
 
 ## Advanced Configuration
 
-### Link your Spark jobs with Airflow task
+### Link your dbt jobs with Airflow tasks
+
+You can monitor your dbt jobs that are running in Airflow by connecting the dbt telemetry with respective Airflow task, using [OpenLineage dbt integration][6].
+
+To see the link between Airflow tasks and dbt jobs, follow those steps:
+
+1. Install openlineage-dbt. MWAA recommends [setting up dbt in the virtualenv.][7]
+
+```shell
+pip3 install openlineage-dbt>=1.33.0
+```
+
+2. Change the dbt invocation to `dbt-ol` (OpenLineage wrapper for dbt)
+
+Optionally, add the --consume-structured-logs flag to view dbt jobs while the command is still running.
```

Comment:
@mobuchowski can we make this required and not optional? Any issue with that? Including this will be a more consistent UX for all customers and I want to minimize options unless they are really necessary.